server/core/storage_test.go

```diff
@@ -146,6 +146,25 @@ func (s *testKVSuite) TestLoadRegions(c *C) {
 	}
 }
 
+func (s *testKVSuite) TestLoadRegionsToCache(c *C) {
+	storage := NewStorage(kv.NewMemoryKV())
+	cache := NewRegionsInfo()
+
+	n := 10
+	regions := mustSaveRegions(c, storage, n)
+	c.Assert(storage.LoadRegionsOnce(cache.SetRegion), IsNil)
+
+	c.Assert(cache.GetRegionCount(), Equals, n)
+	for _, region := range cache.GetMetaRegions() {
+		c.Assert(region, DeepEquals, regions[region.GetId()])
+	}
+
+	n = 20
+	mustSaveRegions(c, storage, n)
+	c.Assert(storage.LoadRegionsOnce(cache.SetRegion), IsNil)
+	c.Assert(cache.GetRegionCount(), Equals, n)
```

Comment:
What are these four lines used for?

Reply:
when we do not useRegionStorage,Test LoadRegionsOnce can load regions from kv(etcd).

Reply:
Oh, how about also testing when `useRegionStorage=1`?

Reply:
in the file region_syncer_test.go. method TestRegionSyncer.
The follower needs to call method 'getRegionsOnce' firstly and then sync the region. What do you think about this
```
	//test All regions have been synchronized to the cache of followerServer
	followerServer := cluster.GetServer(cluster.GetFollower())
	c.Assert(followerServer, NotNil)
	cacheRegions := followerServer.GetServer().GetBasicCluster().GetRegions()
	c.Assert(cacheRegions, HasLen, regionLen)
	for _, region := range cacheRegions {
		c.Assert(followerServer.GetServer().GetBasicCluster().GetRegion(region.GetID()).GetMeta(), DeepEquals, region.GetMeta())
	}
```

Reply:
Actually, what I mean is since you have tested the case which we do not useRegionStorage. It's better to test the case that we enable `useRegionStorage` but the regions only can be loaded once.

Reply:
cc @mantuliu

Reply:
PTAL.@rleungx . Because the useRegionStorage test needs to create the raftCluster,so I add the test in the method TestLoadClusterInfo of the file cluster_test.go.